packages/utils/src/network/decomposeDataUri.ts

```diff
@@ -1,31 +1,47 @@
 import { DATA_URI } from '../const';
 
-/**
- * @memberof PIXI.utils
- */
 export interface DecomposedDataUri {
-    /**
-     * @property {string} Media type, eg. `image`
-     */
     mediaType: string;
-    /**
-     * @property {string} subType Sub type, eg. `png`
-     */
     subType: string;
-    /**
-     * @property {string} charset
-     */
     charset: string;
-    /**
-     * @property {string} encoding Data encoding, eg. `base64`
-     */
     encoding: string;
-    /**
-     * @property {string} data The actual data
-     */
     data: string;
 }
 
+/**
+ * @memberof PIXI.utils
+ * @interface DecomposedDataUri
```

Comment:
Why not just add the properties here?

Reply:
I'm not sure what you mean? I tried adding member like so but this didnt work

```
/**
 * @memberof PIXI.utils
 * @interface DecomposedDataUri
 * @member {string} mediaType type, eg. `image`
 * @member {string} subType Sub type, eg. `png`
 * @member {string} charset type, eg. `image`
 * @member {string} encoding Data encoding, eg. `base64`
 * @member {string} data The actual data
 */
```

I just copied the structure that IPoint implemented for its docs

Reply:
Try `@property`

Reply:
Adding property results in an empty interface

```
/**
 * @memberof PIXI.utils
 * @interface DecomposedDataUri
 * @property {string} mediaType type, eg. `image`
 * @property {string} subType Sub type, eg. `png`
 * @property {string} charset type, eg. `image`
 * @property {string} encoding Data encoding, eg. `base64`
 * @property {string} data The actual data
 */
interface DecomposedDataUri {
}
```

Reply:
Okay. Not use fighting with tsd jsdocs